currency-rates/integrations_tests/currency-rates/handler_test.go

```diff
@@ -0,0 +1,107 @@
+//go:build integration
+
+package currencyrates
```

Comment:
I want you to think about your code in perspective of adjusment. So when you will need to add new routes and new logics to test it will be really hard to test them. Maybe you can somehow separate responsibilty of build test enviroment from actual tests. 
You can try to approach like suite.Suite https://hsleep.medium.com/about-testify-suite-package-ec818a02b8f

Reply:
Thanks, applied suits